doc/stanzas/library.rst

```diff
@@ -52,12 +52,12 @@ order to declare a multi-directory library, you need to use the
   ``dune`` file. This includes ones present in the file system as well as ones
   generated by user rules. You can restrict this list by using a ``(modules
   <modules>)`` field. ``<modules>`` uses the
-  :doc:`concepts/ordered-set-language`, where elements are module names and
+  :doc:`reference/ordered-set-language`, where elements are module names and
   don't need to start with an uppercase letter. For instance, to exclude module
   ``Foo``, use ``(modules (:standard \ foo))``
 
 - ``(libraries <library-dependencies>)`` specifies the library's dependencies.
-  See :doc:`concepts/library-dependencies` for more details.
+  See :doc:`reference/library-dependencies` for more details.
 
 - ``(wrapped <boolean>)`` specifies whether the library modules should be
   available only through the top-level library module, or if they should all be
```

Comment:
Is this toplevel (like the OCaml toplevel) or the top level or something? If the latter, it's fine as it is. If the former, write it as one word: toplevel. It appears to be the latter...just double checking! 😄

Reply:
When you have a library you can specify if it is "wrapped" or not. All this means is that a certain module called the "top-level module" is used as a wrapper around the rest of the library. Say you had 3 modules in your library:
```
secret.ml
my_logic.ml
my_lib.ml
```
Then you could imagine my_lib.ml being the top-level module but it only gives users access to my_logic say. Because you don't want people knowing what secret does.

When a library is unwrapped all the modules are available as "top-level modules".

Reply:
Great! Thanks for explaining.